README.md

```diff
@@ -13,8 +13,48 @@ This textbook gives scientists:
 - a guide to designing parallel processing algorithms to work efficiently with chunked datasets
 - a guide to exporting chunked datasets to other 'tradditional' datasets
 
+## Benchmarking for Zarr
+
+We created a set of benchmarks for writing data to Zarr with a range of different configurations as guidance for the options available when reading and writing 3D imaging data.
+The different parameters were:
```

Comment:
Link to the final report?
e.g. 'The final benchmarking results are summarised into [a short report](https://heftieproject.github.io/zarr-benchmarks/) to help researchers choose the best options for their image datasets'